float32 steering
float32 throttle
string img_path
string depth_path
